client/src/client.rs

```diff
@@ -342,12 +342,12 @@ impl Client {
     ///
     /// # Errors
     /// Fails if sending transaction to peer fails or if it response with error
-    #[log]
```

Comment:
was this removed due to some deadlock issues with it?

Reply:
Yes actually. This PR started by me investigating the problem that @Arjentix idenitified. It turned out to be related to a few others.

Reply:
@appetrosyan have someone else also found this bug with `#[log]`? Am I not a mad man? Cause deadlock in span creating process is really wierd.
If so, maybe we need to spend some time creating issue for the [`tracing` repo](https://github.com/tokio-rs/tracing)? Or search for already openned one

Reply:
You're definitely not going insane. Like i said, the likely culprit is the locking that happens with the global mutable state. I found it as a culprit for a different problem, though.

Reply:
I would recommend you create the issue, and get in touch with them via the Discord, I only managed to reproduce the issue by running `cargo flaky` and seeing a reduction in failed test cases (e.g. `7_peers_1_fault`), you might have a better MWE.